governance.md

```diff
@@ -137,6 +137,13 @@ In order to standardize Special Interest Group efforts, create maximum transpare
 * Participate in release planning meetings and retrospectives, and burndown meetings, as needed
 * Ensure related work happens in a project-owned github org and repository, with code and tests explicitly owned and supported by the SIG, including issue triage, PR reviews, test-failure response, bug fixes, etc. 
 * Use the above forums as the primary means of working, communicating, and collaborating, as opposed to private emails and meetings
+* Represent the SIG for the PM group:
```

Comment:
> So reading this closer, the net takeaway is that someone from each SIG must attend a PM group meeting of some kind at some interval? I had initially read this as meaning that someone from the pm group would participate in each sig and not vice versa?

There's a thread here suggesting the same thing: https://groups.google.com/forum/#!searchin/kubernetes-dev/inverted%7Csort:relevance/kubernetes-dev/pj5q4HjN8E8/u9qe_7NpFwAJ